### PR TITLE
REGD-2 delete unused code

### PR DESCRIPTION
**Pull request by emro**
- Deletes a lot of unused code (tests, test data, images, JS files, style files, etc)
- Adds three Regulome-specific tests and test data
- Help page is now a component

### CODE REVIEW

**Comment by mingjiecn** [comment]
I wasn't able to install the app on Apple m1 pro laptop with macOS Monterey12.0.1
image

**Comment by hitz** [comment]
@mingjiecn emma didn't touch any python or configuration files; so I would expect you get this error on dev as well.  Does regulomedb even run on python3.8?

**Comment by hitz** [comment]
google indicates there are some issues with python3.7 and 3.8 and the "typing" module.

File: src/encoded/static/components/__tests__/regulome-help-test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+
+// Import test component
+import RegulomeHelp from '../regulome_help';
+
+
+describe('Help', () => {
+    describe('Help page', () => {
```
Comment (mingjiecn) [inline src/encoded/static/components/__tests__/regulome-help-test.js:10]: Why do we need two describe?

File: src/encoded/static/components/__tests__/regulome-summary-test.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            const downloadTsv = summary.find('[data-test="download-tsv"]');
+            expect(downloadTsv.text()).toEqual('Download TSV');
+        });
+
+        it('Has summary table', () => {
+            const table = summary.find('.summary-table-hoverable');
+            const tableHeader = table.find('th');
+            expect(tableHeader.at(0).text()).toEqual('Chromosome location');
+        });
+    });
```
Comment (mingjiecn) [inline src/encoded/static/components/__tests__/regulome-summary-test.js:38]: Do we need to unmount summary after tests are done?

**Comment by mingjiecn** [comment]
@hitz google online and found similar results with yours. readme indicate the python version for regulome is Python 3.8.10. Readme also indicates the node version is 10 but for encode the node version is 12. I was using node 12 for installation. I asked around and no one has made it working on apple m1 yet. Talked with @ottojolanki , he suggests Docker Compose for regulome.

**Review by mingjiecn — APPROVED** [review]
wasn't able to install the app with python3.8 and node 10. But I was able to run javascript test using `npm test` and all the tests passed. Since @hitz mentioned nothing changed in configuration then I am ok to approve this PR.

**Pull request merged**